index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lusaka Fashion week</title>
+
+    <!-- Link Lato font from Google fonts -->
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+
+    <!-- Link file for the cocogoose font -->
+    <link rel="stylesheet" href="fonts/cocogoose/Cocogoose.css" />
+
+    <!-- Global style file -->
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Desktop style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="css/desktop.css"
+    />
+    <!-- Mobile style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (max-width: 768px)"
+      href="css/mobile.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="toggle-icon"></div>
+      <div class="menu">
+        <nav class="secondary-menu desktop-padding">
+          <ul>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-facebook.svg"
+                  alt="Facebook small icon"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-twitter.svg"
+                  alt="Twitter small icon"
+                />
+              </a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Profile</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+        <nav class="primary-menu desktop-padding">
+          <a href="/"><img src="images/log02.svg" alt="main logo" /></a>
+          <div class="primary-menu-content">
+            <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Schedule</a></li>
+              <li><a href="#">Tickets</a></li>
+              <li><a href="#">Designers</a></li>
+              <li><a href="#">Updates</a></li>
+            </ul>
+            <button type="button" class="register-menu-btn orange">
+              Register
+            </button>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section id="hero" class="desktop-padding">
+        <div class="desktop-padding">
+          <h2 class="orange">"Hello Fashion Ethusiasts!"</h2>
+          <div class="hero-title cocogoose">
+            The Lusaka Fashion week
+          </div>
```

Comment:
- [ ] Hi, you should have a one `h1` tag on each HTML page. I recommend you convert this `div` to a `h1` tag.
```suggestion
          <h1 class="hero-title cocogoose">
            The Lusaka Fashion week
          </h1>
```